pkgs/by-name/ma/mathemagix/package.nix

```diff
@@ -0,0 +1,45 @@
+{
+  stdenv,
+  lib,
+  fetchsvn,
+  readline,
+  ncurses,
+  bison,
+  libtool,
+  gmp,
+  mpfr,
+}:
+
+stdenv.mkDerivation {
+  pname = "mathemagix";
+  version = "11126";
+
+  src = fetchsvn {
+    url = "https://subversion.renater.fr/anonscm/svn/mmx/";
+    rev = 11126;
+    hash = "sha256-AFnYd5oFg/wgaHPjfZmqXNljEpoFW4h6f3UG+KZauEs=";
+  };
+
+  nativeBuildInputs = [
+    readline
+    ncurses
+    bison
+    libtool
+    gmp
+    mpfr
+  ];
+
+  preConfigure = ''
+    export HOME="$PWD"
+  '';
+
+  configureFlags = [ "--prefix=${placeholder "out"}" ];
+
+  meta = {
+    description = "Mathemagix is a free computer algebra and analysis system. It consists of a high level language with a compiler and a series of mathematical libraries, some of which are written in C++.";
+    homepage = "http://www.mathemagix.org/";
+    license = lib.licenses.gpl3Only;
```

Comment:
Any source? https://subversion.renater.fr/anonscm/svn/mmx/numerix/install-sh and https://subversion.renater.fr/anonscm/svn/mmx/configure for example are not GPL3

Reply:
As far I've seen, all the licenses used in the project are GPL (https://subversion.renater.fr/anonscm/svn/mmx/mmxlight/LICENSE, https://subversion.renater.fr/anonscm/svn/mmx/jorix/LICENSE, https://subversion.renater.fr/anonscm/svn/mmx/caas/LICENSE, etc etc... ).

Reply:
Are we all OK to move on as it is now and refine later on if needed?

Reply:
Yes, all LICENSE files I've seen specify GPL3.

The LICENSE file in the directory of /mmx/numerix/install-sh is GPL3, so I believe install-sh is in fact GPL3.

While /mmx/configure may not directly specify GPL3, it does state
```
# This configure script is free software; the Free Software Foundation
# gives unlimited permission to copy, distribute and modify it.
```

Clearly the author intended for this software to be freely distributed. Getting ahold of academic researchers is often a difficult task. If people are uneasy about a configuration script or two not being marked as GPL3, perhaps there is a different licensing option we can use.

Personally I think this is a non-issue. I think this package is ready to be merged.